Source/WebKit/WebProcess/InjectedBundle/API/APIInjectedBundleBundleClient.h

```diff
@@ -43,10 +43,10 @@ class Client {
 public:
     virtual ~Client() = default;
 
-    virtual void didCreatePage(WebKit::InjectedBundle&, WebKit::WebPage&) { }
-    virtual void willDestroyPage(WebKit::InjectedBundle&, WebKit::WebPage&) { }
-    virtual void didReceiveMessage(WebKit::InjectedBundle&, const WTF::String&, API::Object*) { }
-    virtual void didReceiveMessageToPage(WebKit::InjectedBundle&, WebKit::WebPage&, const WTF::String&, API::Object*) { }
+    virtual void didCreatePage(WebKit::InjectedBundle&, Ref<WebKit::WebPage>&&) { }
```

Comment:
So these can stay taking raw reference: WebKit::WebPage. It's just that the caller has to hav a Ref/RefPtr before calling this function.